### ISSUE TITLE
Wrong link for Github in NuGet repository.

### ISSUE THREAD

**Issue by Palid** [CoraleStudios/Colore#91]
Link in NuGet repository is `https://github.com/Corale/Colore`, while current one is `https://github.com/CoraleStudios/Colore`

**Comment by Sharparam** [comment]
Line `10` in `Corale.Colore/Corale.Colore.nuspec`:

```xml
<projectUrl>https://github.com/Corale/Colore</projectUrl>
                                    ^
```

**Comment by brandonscott** [comment]
Thanks @Palid 

As Adam suggested, we will be sure to update this in 3.0.

**Comment by Sharparam** [comment]
This was fixed in #103
